deploy/LOCAL_DEVELOPMENT_INSTALL.md

```diff
@@ -17,7 +17,7 @@ After installation, run `psql -l` and if there is no user named `postgres`, run
 
 - [redis](https://redis.io/topics/quickstart)
 
-- [node/npm](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm)
+- [node/npm <14](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm).  Note: more recent versions of `node` may not function are not officially supported.
```

Comment:
`node 20` (brew install default) hits [this issue](https://stackoverflow.com/questions/75167770/error-error0308010cdigital-envelope-routinesunsupported-node-js-v19-4-0)

Reply:
😢  can you make a ticket that we should update to a more recent node version? Its low priority, but worth doing at some point, and for now this works

Reply:
Ticket created [here](https://github.com/broadinstitute/seqr/issues/3640)